### ISSUE TITLE
About TCP/IP support

### ISSUE THREAD

**Issue by mrGoodWolf** [mikaku/Fiwix#96]
It might be helpful if the tcp/ip code is out-source to something like lwIP and  then ported to fiwix?

**Comment by mikaku** [comment]
Yes, of course, this is a possibility that I've already thought.

Building a complete network stack in the kernel is not trivial and it could take a lot of time. Instead, using lwIP sounds like a good alternative, as long as the kernel can satisfy all the requirements of lwIP. Also, it's important to keep a good Linux ioctl() API compatibility to be able to port easily the current GNU network tools.

Let me know how can I help you if you want to try this path.

**Comment by mikaku** [comment]
Updates about networking support will be posted on #36.
